tests/panther_mcp_core/test_fastmcp_integration.py

```diff
@@ -1,5 +1,8 @@
+import asyncio
 import os
+import threading
 
+import httpx
 import pytest
 from fastmcp.exceptions import ToolError
 
```

Comment:
this skips this test always unless we run `make integration-test` yeah? Is it worth adding an integration test case to CI somewhere?